backend/database.js

```diff
@@ -51,6 +85,7 @@ sbp('sbp/selectors/register', {
           const entry = await sbp('chelonia/db/getEntry', currentHEAD)
           const json = `"${strToB64(entry.serialize())}"`
           if (currentHEAD !== hash) {
+            const json = `"${strToB64(entry.serialize())}"`
```

Comment:
Indeed this doesn't seem necessary.

Reply:
@snowteamer what's the update on this?

Reply:
This has been simplified and the duplicate line has been removed

Reply:
I see this was done the same way previously, but I'd use `JSON.stringify(foo)` instead of `\`"${foo}"\`` because that'll take care of escaping if there's ever a need to do so.